src/main/config/devices/ios.yaml

```diff
@@ -247,4 +247,15 @@ modifiers:
        minimum: 0
        maximum: 4294967295
        minimumStep: 0
-       maximumStep: 20000
\ No newline at end of file
+       maximumStep: 20000
+# topology changes
+  - oid: ".1.3.6.1.2.1.17.2.4.0"
+    class: "com.oneandone.snmpman.configuration.modifier.CommunityIndexCounter32Modifier"
+    properties:
+      42:
+        1.3.6.1.2.1.17.2.4.0: 150
+      9:
+        1.3.6.1.2.1.17.2.4.0: 120
+vlans:
+  - 20
```

Comment:
that must be '42', otherwise doesen't make any sense... 42 is the answer for everything